### DISCUSSION TITLE
Help, Kade Engine Problems and more.

### DISCUSSION THREAD

**Discussion by PandaFNF** [KadeArchive/Kade-Engine#1236] [Q&A]
source/PlayState.hx:3379: characters 17-32 : Class<webm.WebmPlayer> has no field SKIP_STEP_LIMIT
C:/HaxeToolkit/haxe/lib/polymod/git/polymod/format/XMLMerge.hx:189: characters 10-23 : Warning : This typedef is deprecated in favor of haxe.xml.Access
C:/HaxeToolkit/haxe/lib/polymod/git/polymod/format/XMLMerge.hx:189: characters 30-43 : Warning : This typedef is deprecated in favor of haxe.xml.Access is what it says
if i try go to export folder then everything is there except the application and i tried copying an application of it in there but nope 
it wont use info fro
![proof](https://user-images.githubusercontent.com/87228543/125160119-a39ddd80-e1be-11eb-894b-945b16c73467.png)
m the source please help

**Comment by isakube** [comment]
remove the line from there then its not that hard

**Reply by PandaFNF** [reply]
?
what line?

**Reply by EcoVeni** [reply]
go to playstate.hx go to the line 3379 and delete the line

**Reply by PandaFNF** [reply]
oh my gosh thank you so much
